presto-docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -137,9 +137,7 @@ Property Name                                                 Description
 ``cassandra.load-policy.shuffle-replicas``                    Set to ``true`` to use ``TokenAwarePolicy`` with shuffling of replicas,
                                                               defaults to ``false``.
 
-``cassandra.load-policy.use-white-list``                      Set to ``true`` to use ``WhiteListPolicy``, defaults to ``false``.
-
-``cassandra.load-policy.white-list.addresses``                Comma-separated list of hosts for ``WhiteListPolicy``.
+``cassandra.load-policy.allow-list.addresses``                Comma-separated list of hosts to allow.
```

Comment:
Please change to `cassandra.load-policy.allowed-addresses`

Reply:
✅